uperf-wrapper/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN dnf copr enable ndokos/pbench -y
 RUN dnf install -y --nodocs git python3-pip python3-numpy python3-requests python3-numpy pbench-uperf
 COPY image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
 RUN dnf install -y --nodocs redis --enablerepo=centos8-appstream
-RUN pip3 install --upgrade-strategy=only-if-needed "elasticsearch>=6.0.0,<=7.0.2" pyyaml
+RUN pip3 install --upgrade-strategy=only-if-needed "elasticsearch>=6.0.0,<=7.0.2" pyyaml redis
```

Comment:
So in the line above, we install redis package, but not sure if that'll get us the python library as well. But if this library is still needed, can you please add it to requirements.txt

Reply:
ack